operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -584,7 +584,11 @@ alertmanager:
     maxSkew: 1
     topologyKey: kubernetes.io/hostname
     whenUnsatisfiable: ScheduleAnyway
-
+    minDomains: 1
```

Comment:
from the [docs](https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/) I see that minDomains can only be specific when `whenUnsatisfiable: DoNotSchedule`. Let's make `minDomains` a commented field so that it doesn't change the rendered manifests.